lib/s3routes/routesUtils.ts

```diff
@@ -695,7 +697,9 @@ export function errorHtmlResponse(
     log.trace('sending generic html error page',
         { err });
     setCommonResponseHeaders(corsHeaders, response, log);
-    response.writeHead(err.code, { 'Content-type': 'text/html' });
+    /** AWS uses old HTTP/1.0 statusMessage for 302 Found */
```

Comment:
Do we care about the message returned? I don't think it would cause compatibility issues to use the newer standard message instead of the deprecated one. Applications normally shouldn't rely on the message for their behavior, only the code, the message should be informational only.

Reply:
This `statusMessage` is used in the response body html. I used it to match exactly what AWS does, but I can ignore it

Reply:
I've removed the override